firebase-perf/src/main/java/com/google/firebase/perf/util/Timer.java

```diff
@@ -23,20 +23,20 @@
 /** A Timer class provides both wall-clock (epoch) time and high resolution time (nano time). */
 public class Timer implements Parcelable {
   /** Wall-clock time or epoch time in microseconds, */
-  private long mTime;
+  private long time;
```

Comment:
`timeInMicros`? According to your suggestions in the ticket, all variables should have contextual names.

Reply:
Done!